cmd/gh/main.go

```diff
@@ -226,7 +226,7 @@ func mainRun() exitCode {
 			fmt.Fprintln(stderr, "Try authenticating with:  gh auth login")
 		} else if strings.Contains(err.Error(), "Resource protected by organization SAML enforcement") {
 			fmt.Fprintln(stderr, "Try re-authenticating with:  gh auth refresh")
-		} else if msg := httpErr.ScopesSuggestion(); msg != "" {
+		} else if msg := httpErr.ScopesSuggestion(); msg != "" && httpErr.StatusCode != 422 {
```

Comment:
Could you move the "not 422" check into ScopesSuggestion function itself? It's where it performs the check that this should be done for HTTP 4xx in the first place. Thank you!